ports/esp32/modmachine.c

```diff
@@ -90,7 +90,7 @@ typedef enum {
 
 static bool is_soft_reset = 0;
 
-#if CONFIG_IDF_TARGET_ESP32C3
+#if CONFIG_IDF_TARGET_ESP32C3 || CONFIG_IDF_TARGET_ESP32C6
```

Comment:
As this is just an extern declaration (for a private header) then suggest removing the ifdef entirely. Maybe put a comment about this being declared in a private header...